src/test/mochitest/browser_dbg-console-map-bindings.js

```diff
@@ -0,0 +1,45 @@
+// Return a promise with a reference to jsterm, opening the split
+// console if necessary.  This cleans up the split console pref so
+// it won't pollute other tests.
+function getSplitConsole(dbg) {
+  const { toolbox, win } = dbg;
+
+  if (!win) {
+    win = toolbox.win;
+  }
+
+  if (!toolbox.splitConsole) {
+    pressKey(dbg, "Escape");
+  }
+
+  return new Promise(resolve => {
```

Comment:
I know it was copy-pasted, but this antipattern always just jumps out at me.

Reply:
hmm, are you suggesting? I think this would be nicer :)

```js
async function getSplitConsole(dbg) {
  // before
  await toolbox.getPanelWhenReady("webconsole");
  // after
  return jsterm
}
```

Reply:
Yep! Or at a minimum
```
return toolbox.getPanelWhenReady("webconsole").then(() => {
  ok(toolbox.splitConsole, "Split console is shown.");
  return toolbox.getPanel("webconsole").hud.jsterm;
});
```
without the `new Promise`.

Reply:
oh ha